### ISSUE TITLE
learning rate declining faster than straight line

### ISSUE THREAD

**Issue by mikowals** [rinuboney/ladder#2]
To get a straight line decline, the learning rate update should be:

    learning_rate = starter_learning_rate * ratio

Also, I don't think it is good practice to assign a tf.Variable() using  `=`.  I think creating a learning rate placeholder and then setting the learning rate in the feed_dict for training is better.

**Comment by rinuboney** [comment]
I think using ```assign``` is better than using ```=``` and more elegant than passing the learning rate explicitly:
```
sess.run(learning_rate.assign(starter_learning_rate * ratio))
```
